rust2vec/src/text.rs

```diff
@@ -172,7 +172,11 @@ where
         }
     }
 
-    Ok(Embeddings::new(SimpleVocab::new(words), NdArray(matrix)))
+    Ok(Embeddings::new(
+        None,
```

Comment:
Not sure if wanted, but we can infer the vocab size and dimensions from embeddings stored in text format.